bpf/lib/lb.h

```diff
@@ -1225,7 +1225,7 @@ struct lb4_service *lb4_lookup_service(struct lb4_key *key,
 	key->backend_slot = 0;
```

Comment:
In the case of socket-LB being disabled (e.g., when `--kube-proxy-replacement=disabled`), the bpf_lxc does the E/W LB for Pods running in non-host netns.  In this case, the `lb{4,6}_lookup_service()` is called in bpf_lxc.c for egressing pkts.

I'm wondering how the internal traffic policy is going to be handled for such a case? Don't we need to switch the scope above to `LB_LOOKUP_SCOPE_INT`?

Reply:
Looks like right now we only do this in bpf_lxc for kpr enabled: `lb{4,6}_lookup_service(&key, is_defined(ENABLE_NODEPORT),  ...)`. Agree that we should call it unconditionally with `lb{4,6}_lookup_service(&key, true,  ...)` and test whether for ClusterIP it matches also kube-proxy behavior.